stumpy/floss.py

```diff
@@ -429,6 +429,7 @@ def __init__(
         self._n = self._T.shape[0]
         self._last_idx = self._n - self._m + 1  # Depends on the changing length of `T`
         self._n_appended = 0
+        self._finite_mask = np.full(self._n, True)
```

Comment:
In other parts of the code, we have a similar mask that is called `T_subseq_isfinite` that basically marks where a subsequence is "illegal". It is computed by doing:

```
T_subseq_isfinite = np.all(np.isfinite(core.rolling_window(T, m)), axis=1)
```

Would you mind naming this attribute `self._T_isfinite`? I think it may be (ever so) slightly more obvious than `_finite_mask`. So, `self._T_isfinite` is for each element of `T` and `T_subseq_isfinite` is for each subsequence of `T`.

I'm wondering if we can leverage `self._T_isfinite` AFTER we compute `mass` by simply doing something similar to `T_subseq_isfinite` like:

```
D = core.mass(Q, self._T, M_T, Σ_T)
D[zone_start:] = np.inf
T_subseq_isfinite = np.all(core.rolling_window(T_isfinite, m), axis=1)
D[T_subseq_isfinite] = np.inf
```

I believe that `T_subseq_isfinite` should have the same length as `D` in this case but I could be wrong. Please let me know if that makes any sense.  Just thinking off the top of my head here :)

Reply:
Great points, Sean.  I'll make the changes to stay consistent with the rest of the code base.

Reply:
Thank you! Let me know if you need to bounce anything off of me. I am here to help!